src/operations.jl

```diff
@@ -1,18 +1,23 @@
-struct TransformsRow{T, F}
+struct TransformsRow{T, F} <: AbstractRow
     row::T
     funcs::F
 end
 
-Base.getproperty(row::TransformsRow, nm::Symbol) = (getfunc(row, getfield(row, 2), nm))(getproperty(getfield(row, 1), nm))
-Base.propertynames(row::TransformsRow) = propertynames(getfield(row, 1))
+getrow(r::TransformsRow) = getfield(r, :row)
+getfuncs(r::TransformsRow) = getfield(r, :funcs)
 
-struct Transforms{C, T, F}
+getcolumn(row::TransformsRow, nm::Symbol) = (getfunc(row, getfuncs(row), nm))(getcolumn(getrow(row), nm))
+getcolumn(row::TransformsRow, i::Int) = (getfunc(row, getfuncs(row), i))(getcolumn(getrow(row), i))
+columnnames(row::TransformsRow) = columnnames(getrow(row))
+
+struct Transforms{C, T, F} <: AbstractColumns
```

Comment:
Before 1.0 should we take the chance to rename this ? (Or to rename the other?)

Reply:
Or remove it entirely. I know @bkamins and @nalimilan have been hammering on the select/transform API in DataFrames, so if there's a way we could generalize their excellent design work into a generic Tables.jl API, I think it'd be a great starting point for a TablesOperations.jl package or something.

Reply:
Good idea.
Deleting things before 1.0 is a good way to free up real estate for redevelopment.

Reply:
Maybe just mention in the docs that this isn't part of the API, and ensure it's unexported? Then we can decide what to do in the long run.